exporters/trace/jaeger/jaeger_test.go

```diff
@@ -930,10 +944,14 @@ func TestNewExporterPipelineWithOptions(t *testing.T) {
 	assert.True(t, span.SpanContext().IsValid())
 
 	batchesUploaded := testCollector.batchesUploaded
-	assert.True(t, len(batchesUploaded) == 1)
+	assert.Len(t, batchesUploaded, 1)
```

Comment:
Using `require` when checking for length here an in the other changes will ensure that the subsequent index will not panic and hide this error when the test is run.

```suggestion
	require.Len(t, batchesUploaded, 1)
```
```suggestion
	assert.Len(t, batchesUploaded, 1)
```